app/services/test_application_creation_service.rb

```diff
@@ -1,13 +1,6 @@
 class TestApplicationCreationService
   APPLICATION_TEST_TRAITS = %i[
-    at_initiated
     at_checking_applicant_details
-    at_checking_passported_answers
-    at_applicant_details_checked
-    awaiting_applicant
-    applicant_entering_means
-    provider_entering_merits
-    at_checking_merits_answers
```

Comment:
I might be missing something but I'm not sure I understand why these are being removed? It seems like it's removing most of the purpose of this class.

Also it doesn't look like `TestApplicationCreationService` is used by the geckoboard tests, so should these changes even be part of this PR?

Reply:
sorry that was leftover stuff. i'll remove all changes to that file